aspnetcore/fundamentals/servers/kestrel.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Kestrel server
+author: tdykstra
+ms.author: tdykstra
+manager: wpickett
+ms.date: 10/27/2016
+ms.topic: article
+ms.assetid: 560bd66f-7dd0-4e68-b5fb-f31477e4b785
+ms.prod: aspnet-core
+uid: fundamentals/servers/kestrel
+---
+# Kestrel server for ASP.NET Core
+
+By [Tom Dykstra](http://github.com/tdykstra), [Chris Ross](https://github.com/Tratcher), and [Stephen Halter](https://twitter.com/halter73)
+
+Kestrel is a cross-platform [web server for ASP.NET Core](overview.md) based on [libuv](https://github.com/libuv/libuv), a cross-platform asynchronous I/O library. Kestrel is the web server that is included by default in ASP.NET Core new-project templates. 
```

Comment:
Used as adjective - https://en.wikipedia.org/wiki/Compound_modifier
But probably not essential in this context